drivers/uart1w/uart1w.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2017 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     drivers_uart1w
+ * @{
+ *
+ * @file
+ * @brief       Driver implementation for 1-wire UART devices
+ *
+ * @author      Loïc Dauphin <loic.dauphin@inria.fr>
+ *
+ * @}
+ */
+
+#include "uart1w.h"
+
+#include "periph/uart.h"
+#include "periph/gpio.h"
+#include "xtimer.h"
+
+#define IS_SET(dir)   ((dir % 2) == 1)
```

Comment:
With compiler optimisation, it is supposed to be the same.
But, yes, I wondered which was better when writing it, and if you think the & is better, let's do that.